index.js

```diff
@@ -119,6 +119,120 @@ function closeModal(i) {
   document.querySelector(`.modal-${i}`).style.display = 'none';
 }
 
+// work section 1.
+
+const modalsOb = [
+  {
+    modalTitl: 'Components website',
+    modalLis: ['css', 'html', 'Bootstrap', 'Ruby'],
+    modalIm: './images/Img Placeholder.png',
+    modalTex:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  }];
+const work = document.querySelector('.description-med');
+
+modalsOb.forEach((o) => {
+  let techStack = '';
+
+  for (let i = 0; i < o.modalLis.length; i += 1) techStack += `<li>${o.modalLis[i]}</li>`;
+
+  work.innerHTML = `
+  <div class="portfolio" id="portfolio">
+  <div>
+  <h2 class="myrr" id="myrr">My Recent Work</h2>
+  <img src="./images/Vecto1.png" alt="blackline" class="blackline">
+</div>
+<div class="img-pp"> <img src="${o.modalIm}"></div>
+<div class="description" id="description">
+  <h2 class="multi">${o.modalTitl}</h2>
+  <p class="daily">${o.modalTex}</p>
+  <ul class="languages">
+          ${techStack} </ul>
+  <button type="button" onclick="showModal(1);" class="but1">See project</button>
+</div>
+<div class="title-med">
+  <div class="img-med"><img src="./images/Img Placeholder1.png"></div>
+  <div class="img-medm"><img src="${o.modalIm}"></div>
+  <div class="title-tree">
+      <h2 class="multi">${o.modalTitl}</h2>
+      <p class="daily">${o.modalTex}</p>
+      <ul class="languages">
+      ${techStack}
+      </ul>
+      <button type="button" onclick="showModal(1);" class="but2">See project</button>
+  </div>
+</div>     
+</div>
+ 
+`;
+});
+
+// WORK SECTION 2
+const modalsO = [
+
+  {
+    modalTitle: 'Data Dashboard Healthcare',
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 review that the title on the modal page corresponds to the card selected, Please fix the titles n the modal pages for a better user interface.